apps/openchallenges/mcp-server/project.json

```diff
@@ -43,6 +50,28 @@
         "cwd": "{projectRoot}"
       },
       "dependsOn": ["^install"]
+    },
+    "serve-detach": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "docker/openchallenges/serve-detach.sh {projectName}"
+      },
+      "dependsOn": []
+    },
+    "build-image-base": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "./gradlew bootBuildImage",
+        "cwd": "{projectRoot}"
+      },
+      "dependsOn": ["^install"]
+    },
+    "scan-image": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "trivy image ghcr.io/sage-bionetworks/{projectName}:local --quiet",
+        "color": true
+      }
     }
   },
   "tags": ["type:service", "scope:backend", "language:java", "package-manager:gradle"],
```

Comment:
@tschaffter  do I need to add the java api client to `implicitDependencies`?

Reply:
Not until it's being used by the MCP server.